### PR TITLE
Release prep

### PR DESCRIPTION
**Pull request by Emiller88**
General release prep. Only thing left to do is use https://www.codenamegenerator.com/, but I don't know what settings to use.

### CODE REVIEW

**Review by drpatelh — APPROVED** [review]
Looks great 🚀 Change the link to your GitHub handle in the credits.

File: README.md [hunk 1]
```diff
@@ -82,11 +66,14 @@ The nf-core/demultiplex pipeline comes with documentation about the pipeline [us
 
 The nf-core/demultiplex pipeline was written by Chelsea Sawyer from The Bioinformatics & Biostatistics Group for use at The Francis Crick Institute, London.
 
-Many thanks to others who have helped out along the way too, including (but not limited to): [`@ChristopherBarrington`](https://github.com/ChristopherBarrington), [`@drpatelh`](https://github.com/drpatelh), [`@danielecook`](https://github.com/danielecook), [`@escudem`](https://github.com/escudem), [`@crickbabs`](https://github.com/crickbabs)
+The pipeline was re-written in Nextflow DSL2 and is primarily maintained by Matthias De Smet([@matthdsm](https://github.com/matthdsm)) from [Center For Medical Genetics Ghent](https://github.com/CenterForMedicalGeneticsGhent) and Edmund Miller([@emiller88](https://github.com/drpatelh)) from [Element Biosciences](https://www.elementbiosciences.com/)
```
Comment (matthdsm) [inline README.md:69]:
```suggestion
The pipeline was re-written in Nextflow DSL2 and is primarily maintained by Matthias De Smet([@matthdsm](https://github.com/matthdsm)) from [Center For Medical Genetics Ghent](https://github.com/CenterForMedicalGeneticsGhent) and Edmund Miller([@emiller88](https://github.com/emiller88)) from [Element Biosciences](https://www.elementbiosciences.com/)
```

**Comment by matthdsm** [comment]
@Emiller88, shouldn't https://github.com/nf-core/demultiplex/blob/release-prep/tests/b2fq-samplesheet.csv be moved to `assets/inputs` with the other samplesheets (or the other way around)? That way we don't have multiple locations for the test inputs.

**Comment by matthdsm** [comment]
No further comments! Seems good to go.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Emiller88) [reply README.md]: Copy-pasting, isn't my strong suite, apparently. :upside_down_face:

**Pull request merged**